package.json

```diff
@@ -155,34 +155,34 @@
     "jsdom-global": "3.0.2",
     "lodash.merge": "4.6.2",
     "merge-stream": "2.0.0",
-    "mocha": "10.4.0",
+    "mocha": "10.7.0",
     "mq-polyfill": "1.1.8",
     "normalize.css": "8.0.1",
     "path": "0.12.7",
-    "postcss": "8.4.38",
+    "postcss": "8.4.40",
     "postcss-csso": "6.0.1",
     "postcss-discard-comments": "6.0.2",
     "postcss-import": "15.1.0",
     "postcss-loader": "7.3.3",
-    "postcss-preset-env": "9.5.11",
-    "prettier": "3.2.5",
+    "postcss-preset-env": "9.6.0",
+    "prettier": "3.3.3",
```

Comment:
**issue**: This is failing silently for `npm run prettier:html`.

```sh

> @uswds/uswds@3.8.1 prettier:html
> npx prettier --write './html-templates/**/**/*.html'

```

Seeing the same in `develop` branch too.

![image](https://github.com/user-attachments/assets/9d383871-e8d7-4a5e-8420-4be312472f21)

Reply:
GREAT catch. Looks like starting with Prettier `3.x.x`, prettier uses our `.gitignore` to decide what files to ignore as well.

[Prettier changelog note about this and how to replicate previous behavior →](https://prettier.io/blog/2023/07/05/3.0.0.html#ignore-gitignored-files-by-default-14731httpsgithubcomprettierprettierpull14731-by-fiskerhttpsgithubcomfisker)

I added the `--ignore-path=.prettierignore` flag to our prettier scripts to maintain our previous ignores. Luckily, we don't have any inadvertent changes. The script now appropriately formats the `html-templates`

```diff
- "prettier:html": "npx prettier --write './html-templates/**/**/*.html'",
- "prettier:js": "npx prettier --write './packages/**/**/*.js'",
- "prettier:sass": "npx prettier --write './packages/**/**/*.scss'",
+ "prettier:html": "npx prettier --write './html-templates/**/**/*.html' --ignore-path=.prettierignore",
+ "prettier:js": "npx prettier --write './packages/**/**/*.js' --ignore-path=.prettierignore",
+ "prettier:sass": "npx prettier --write './packages/**/**/*.scss' --ignore-path=.prettierignore",
```

Resolved in [74e8c59](https://github.com/uswds/uswds/pull/5972/commits/74e8c597d2fd8705ac7e945699b5bf1fb6ac022b)